app/src/ui/lib/list/list.tsx

```diff
@@ -225,6 +225,8 @@ interface IListProps {
    * where to scroll do on rendering of the list.
    */
   readonly setScrollTop?: number
+
+  readonly forceGridUpdate?: boolean
```

Comment:
I think we should try to use the `invalidationProps` prop to simulate this if we can. Just add a `shasNotInDiffHighlighted` prop to the list itself, and then pass it to the `invalidationProps` so that when that prop changes it will update the grid too. Did you try that? Maybe I'm missing something 😳

Reply:
Yay! 🎉